panel/template/fast/list/fast_list_template.html

```diff
@@ -84,7 +84,7 @@
 <fast-design-system-provider id="body-design-provider" use-defaults>
   <div id="container">
     <fast-design-system-provider id="header-design-provider">
-      <nav id="header" {{ 'class="shadow"' if style.shadow else '' }} >
+      <nav id="header" {% if header_max_width %}style="margin-left: auto; margin-right: auto; max-width: {{ header_max_width }}"{% endif %} >
```

Comment:
Would indeed prefer a CSS class.

Reply:
Should I also migrate the `main_max_width` or perhaps just header_max_width?